test/kubemark/resources/manifests/addons/kubemark-rbac-bindings/heapster-binding.yaml

```diff
@@ -0,0 +1,16 @@
+# This is the role binding for the kubemark heapster.
+# TODO: Make this role weaker but still sufficiently powerful.
```

Comment:
Does this TODO still hold true now that the heapster role is granted only the permissions it needs?